### DISCUSSION TITLE
Lines Around My Bags

### DISCUSSION THREAD

**Discussion by torkbjo** [SFX-WoW/Masque#173] [Issues] [answered]
Hi, I keep getting green and purple lines around my bags 24/7. Whenever I go in and out of battle, reload UI, logging in, teleport to a new place etc. After changing something in Masque, the lines disappears, but then they come back. Is there a fix for this is this a known bug? I just have Dominos, Masque and Masque Renaitre installed.

![screenshot_2_lines](https://user-images.githubusercontent.com/34964081/34455624-67501420-ed83-11e7-86c5-7b988b7067b3.png)

**Comment by StormFX** [comment, marked as answer]
This is not an issue with Masque. It looks Dominos is showing the item quality on the border color for the bags.
